src/ads-commons.ts

```diff
@@ -841,15 +841,15 @@ export const ADS_RCP_METHOD_FLAGS = {
 
 /**
  * ADS RCP method parameter flags
- * 
+ *
  * Source: TwinCAT.Ads.dll By Beckhoff
  */
 export const ADS_RCP_METHOD_PARAM_FLAGS = {
   /**  Input Parameter (ADSMETHODPARAFLAG_IN) */
   In: 1,
   /**  Output Parameter (ADSMETHODPARAFLAG_OUT) */
   Out: 2,
-  /**  
+  /**
```

Comment:
Just realised that my editor eagerly trimmed trailing whitespace in the process.